graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/alarms/AlertCondition.java

```diff
@@ -37,15 +35,6 @@
 public interface AlertCondition {
     String getDescription();
 
-    /**
-     * The limited list of internal message objects that matched the alert.
-     * @see org.graylog2.plugin.alarms.AlertCondition.CheckResult#getMatchingMessages()
-     * @return list of Message objects
-     */
-    @Deprecated
-    @JsonIgnore
-    List<Message> getSearchHits();
```

Comment:
Although the method has been marked as deprecated, this is a breaking change which affects all existing alarm callbacks (and there are actually some plugins providing those), so I would not rename that method in the Graylog 1.x version line.